src/main/java/mezz/jei/gui/ghost/GhostIngredientDrag.java

```diff
@@ -75,22 +78,35 @@ public void drawItem(Minecraft minecraft, PoseStack poseStack, int mouseX, int m
 				float percentOfDim = Math.min(1, distanceSq / (float) screenDim);
 				lineWidth = 1 + ((1 - (percentOfDim)) * 3);
 			}
-			GL11.glDisable(GL11.GL_TEXTURE_2D);
+
+			RenderSystem.disableTexture();
 			RenderSystem.disableDepthTest();
+			RenderSystem.depthMask(false);
+
+			var oldShader = RenderSystem.getShader();
```

Comment:
I am not sure what mezz's stance on this is, but figured I would point it out in case mezz feels strongly one way or the other and just missed it. Glancing at the JEI codebase it seems the three additions of `var` here would be the only uses of `var` in JEI.

Reply:
Good to point out. 
I think `var` is good, I'm happy to move everything to use it.
I'm hoping Mojang's mappings remain relatively stable, for example it could be confusing for porting here if the types change and we are using `var`, since we wouldn't have much information about what type it used to be.